### ISSUE TITLE
After tapping someone's name to private-message them, it's hard to tap the new "tab" at the bottom, to enter the PM screen

### ISSUE THREAD

**Issue by dholbert** [nickdesaulniers/fxos-irc#93]
My STR:
1. Open Firesea IRC app on Sony Xperia Z3 phone, in Firefox OS (3.0)
2. Connect, w/ default values
3. When someone says something, tap their username.
   (Note that a private message tab appears for them at bottom of screen)
4. Try to tap the private message tab.

EXPECTED RESULTS: Switch to private message tab.
ACTUAL RESULTS: Nothing happens, or the keyboard comes up. After repeated mashing of the screen area around the tab, I can eventually get it to trigger, but it takes a while. Something is really whack about the touch events for that tab.

**Comment by dholbert** [comment]
I get similar results when I try to tap the "Setup" tab, when I'm in an IRC channel tab.

**Comment by nickdesaulniers** [comment]
Indeed, we've seen Flames with really poor ability to detect touches near the edge of the screen.  I wonder if expanding the height of the tabs help.  @dholbert , can you remote debug via webIDE, use the inspector to add to the height of tabs, and see if that helps?

**Comment by dholbert** [comment]
> Indeed, we've seen Flames with really poor ability to detect touches near the edge of the screen.

Yup, I had one of those. :) I don't think that's the problem here, though, because:
- I'm encountering this on a Xperia Z3, not a flame.
- the touch isn't discarded (as it was on my flame w/ that dead-spot problem) - it's just mis-targeted. When I'm on the chat screen, it frequently targets the text field & brings up the keyboard (but I'm sure I tapped the tab, not the text field).
- once I've successfully switched tabs once, it works great. It's only busted the first time (or maybe the first time after a new tab has been opened)

> @dholbert , can you remote debug via webIDE, use the inspector to
> add to the height of tabs, and see if that helps?

That does indeed help. (I gave `<x-tabbar>` a height of 120px.) I can still reproduce the problem when tapping near the top of the tabs, though (before I've successfully switched).
